freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_replica_install.sh

```diff
@@ -33,6 +32,17 @@ ipa-client-install \
   --ssh-trust-dns \
   --no-ntp
 
+CLOUD_NAMESERVER=$(grep -Ev '^#|^;' /etc/resolv.conf.orig | grep nameserver | awk '{print $2}')
```

Comment:
this is breaking existing functionality:
```
# cat test
nameserver 1.2.3.4
nameserver 5.4.3.2
```
old output:
```
# grep -Ev '^#|^;' test | grep nameserver | awk '{print "--forwarder " $2}'
--forwarder 1.2.3.4
--forwarder 5.4.3.2
```
new output:
```
grep -Ev '^#|^;' test | grep nameserver | awk '{print $2}'
1.2.3.4
5.4.3.2
```

so the new one won't handle when there is multiple nameserver on the network

Reply:
Got it. Will fix it. I missed the awk change.

Reply:
Sorry about this, resolved the issue now.